tests/foreman/ui/test_org.py

```diff
@@ -322,41 +347,24 @@ def test_negative_update(self, new_name):
 
         @assert: Organization name is not updated
 
-        @BZ: 1131469
-
         """
         org_name = gen_string('alpha')
         with Session(self.browser) as session:
             make_org(session, org_name=org_name)
             self.assertIsNotNone(self.org.search(org_name))
-            self.org.update(org_name, new_name=new_name)
-            error = session.nav.wait_until_element(
-                common_locators['name_haserror'])
-            self.assertIsNotNone(error)
+            for new_name in invalid_names_list():
+                with self.subTest(new_name):
+                    self.org.update(org_name, new_name=new_name)
+                    error = session.nav.wait_until_element(
+                        common_locators['name_haserror'])
+                    self.assertIsNotNone(error)
 
     # Miscellaneous
 
-    @skip_if_bug_open('bugzilla', 1131469)
-    @data(*generate_strings_list())
-    def test_search_key(self, org_name):
```

Comment:
Any particular reason for deleting this test? I know there's tons of calls here that perform a search but do you think this specific test with its randomness and range of string types is redundant?